src/BlobField/style-blobs.css

```diff
@@ -1,22 +1,22 @@
-.wrapper {
+.blobs-wrapper {
```

Comment:
Changing from SASS to pure CSS

Reply:
did you try this with the modules in place? I'm wondering if it was actually sass that was breaking it.

Reply:
I'm gonna give it a try and report back.

Reply:
No success :( 

I found:

- https://github.com/stereobooster/react-snap/issues/437 which seems to be exactly our issue  as "this only occurs when running the production build".
- https://github.com/stereobooster/react-snap/issues/214#issuecomment-437877699
- https://github.com/stereobooster/react-snap/issues/432
- https://github.com/stereobooster/react-snap/issues/99 (`window.snapSaveState`?)

There's also the following in my console during dev preview:
```
Warning: Expected server HTML to contain a matching <div> in <div>.
    in div (at App/index.js:10)
    in Router (created by BrowserRouter)
    in BrowserRouter (at App/index.js:9)
    in Unknown (at src/index.js:17)
    in StrictMode (at src/index.js:16)
console.<computed> @ index.js:1
```
Wondering if this has to do with that lingering `","`  but figuring it out is way over my head and my react knowledge.

Reply:
ok. i can take a look after the lecture tonight

Reply:
Hey managed to narrow down the issue to `PageHeader` component's outermost `div`. Removing the `className` on this element fixes the problem. But no matter how hard I tried to add the class back, be it through pure css, having the css in the public folder and linking in html, sass, etc, it kept messing up the bg titles after returning to homepage from an artist page. I hope this helps. 

https://github.com/blerchin/dma-20-mfa-show/commit/7767417d3a2dc8bf9c8fecc5ecf6f050c2cdd40e